### ISSUE TITLE
Support FZF_HISTORY_OPTS

### ISSUE THREAD

**Issue by degrix** [PatrickF1/fzf.fish#87]
Hello! 

Great plugin so far, but it would be nice if the functionality provided by the `FZF_REVERSE_ISEARCH_OPTS` in the jethrokuan/fzf were supported. See: https://github.com/jethrokuan/fzf/blob/master/functions/__fzf_reverse_isearch.fish.  I have a (probably) niche use case where I like to have a custom preview window for long history commands when searching through my history but don't necessarily want to export that functionality in the `FZF_DEFAULT_OPTS`. Having a variable to tweak to add functionality to the `__fzf_search_history` would be great.

e.g. `fzf --read0 --tiebreak=index --query=(commandline)` -> `fzf --read0 --tiebreak=index $FZF_HISTORY_OPTS  --query=(commandline)`

**Comment by PatrickF1** [comment]
Hi @degrix! Thank you for your kind words. If you don't mind--I have a shameless ask--could you star this repo?

Wow that is indeed a niche case! You got me interested in what you're doing, maybe I'd want to copy it. What do you mean preview long commands? What would you set for FZF_HISTORY_OPTS?

Anyway, I'm currently leaning against it. The amount of extra work to document it and test it is pretty annoying. However, it shouldn't  be hard for you, the user, to hack it together. May I suggest changing your key binding to this in the meantime?
```
bind \cr '__fzf_search_history' 'env FZF_DEFAULT_OPTS="$FZF_DEFAULT_OPTS $FZF_HISTORY_OPTS"'
```
Basically, all it does is prepend your desired options to the FZF_DEFAULT_OPTS environment variable before executing that one command.

**Comment by degrix** [comment]
My typical workflow involves making use of a lot of similar, but long, curl commands in my history that typically vary only slightly in their content.  Picking which one is correct without being able to see the entire command is difficult without a preview window.  I've attached a screen grab as an example of the preview window with a long command. The preview window is always visible even for short commands, be invaluable for long commands.  

![Screen Shot 2021-01-26 at 9 35 49 AM](https://user-images.githubusercontent.com/12463028/105874582-fa6d7d00-5fb9-11eb-840f-ecd9996f8333.png)

In the past, I've been bitten once or twice by long commands in the history and choosing the wrong one.  Most of the time that's benign, but if the command is destructive by nature, and happens to be tacked onto the end of a chain of commands, it may result in unwanted behavior.

In any case, I've starred the repo and that bind allows me to workaround this issue that I have, so thanks for that!
